plotly/figure_factory/_facet_grid.py

```diff
@@ -25,6 +25,7 @@
 THRES_FOR_FLIPPED_FACET_TITLES = 10
 GRID_WIDTH = 1
 
+VALID_TRACE_TYPES = ['scatter', 'scattergl', 'histogram', 'bar', 'box']
```

Comment:
are we planning on adding `histogram2d` as well (https://github.com/plotly/plotly.py/pull/776#pullrequestreview-46325756) ?